definitions/contested/json/CaseEventToComplexTypes/CaseEventToComplexTypes-manageScannedDocs.json

```diff
@@ -4,64 +4,97 @@
     "ID": "FR_manageScannedDocs",
     "CaseEventID": "FR_manageScannedDocs",
     "CaseFieldID": "manageScannedDocumentCollection",
-    "ListElementCode": "caseDocuments",
+    "ListElementCode": "selectForUpdate",
```

Comment:
How is this element code change going to work out with existing data?

Reply:
It's a new field that is populated in the about to start event. All the data in manageScannedDocumentCollection is created from scratch when Manage scanned documents event is run.